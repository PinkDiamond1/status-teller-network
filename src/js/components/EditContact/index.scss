.infos-and-warnings {
  .info {
    font-size: 12px;
  }
}
