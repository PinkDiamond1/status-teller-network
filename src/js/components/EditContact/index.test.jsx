import React from "react";
import { shallow } from "enzyme";

import EditContact from "./index";

describe('EditContact', () => {
  it('should render correctly', () => {
    const component = shallow(<EditContact changeNickname={() => undefined} changeContactCode={() => undefined}
                                           resolveENSName={() => undefined} ensError="" nickname={"nickname"} statusContactCode={"contractCode"} />);

    expect(component).toMatchSnapshot();
  });
});
