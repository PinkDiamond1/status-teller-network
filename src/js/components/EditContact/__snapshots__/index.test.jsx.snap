// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditContact should render correctly 1`] = `
<Fragment>
  <h2 />
  <p />
  <Form(t)>
    <FormGroup
      tag="div"
    >
      <Label
        className="text-dark"
        for="nickname"
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Set nickname
      </Label>
      <Control(s)
        className="form-control"
        id="nickname"
        name="nickname"
        onChange={[Function]}
        type="text"
        validations={
          Array [
            [Function],
          ]
        }
      />
    </FormGroup>
    <FormGroup
      tag="div"
    >
      <Label
        className="text-dark"
        for="contactCode"
        tag="label"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        Status contact code or Status ENS name
      </Label>
      <Control(s)
        className="form-control"
        id="contactCode"
        name="contactCode"
        onChange={[Function]}
        type="text"
        validations={
          Array [
            [Function],
            [Function],
          ]
        }
      />
    </FormGroup>
  </Form(t)>
</Fragment>
`;
